src/organizations.js

```diff
@@ -1,12 +1,42 @@
 const { request } = require('./http');
 
 
+const create = (data, opts) => request(opts, {
```

Comment:
I'm wondering if instead of just taking a data blob, if we take the required data as the command line arguments (eg. `(name, archived=false, opts)`), to provide some verification client side, to avoid mistakes when using the client.. Or how is this usually done?